src/main/java/com/example/bookstore/dto/BookSearchParametersDto.java

```diff
@@ -1,12 +1,16 @@
 package com.example.bookstore.dto;
 
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Positive;
 import java.util.List;
 
 public record BookSearchParametersDto(
         List<String> title,
         List<String> author,
         List<String> isbn,
+        @Positive @Digits(integer = 8, fraction = 2)
```

Comment:
Positive annotaiton works only with numeric values

Reply:
You must be kidding me.